package.json

```diff
@@ -21,8 +21,7 @@
     "parse-formdata": "^1.0.2",
     "standard": "^10.0.3",
     "tap-spec": "^4.1.1",
-    "tape": "^4.8.0",
-    "webpack": "^3.10.0"
+    "tape": "^4.8.0"
```

Comment:
Is it worth listing `webpack` in `peerDependencies `?

```
"peerDependencies": {
    "webpack": "^3 || ^4"
  },
```

Reply:
I'm not sure… I've only ever seen `peerDependencies` cause issues (or just annoying warnings). What happens when Webpack v5 comes out but there are no updates to the plugin interface? I guess we could put `"webpack": ">=3` but I just don't see much value in it. If you're attempting to use a webpack plugin and webpack isn't installed, it's going to be pretty obvious.

Reply:
@wordofchristian do you have an opinion on this?

Reply:
having adding `"webpack": ">=3` to the peer dependencies seems like a good idea and consistent with what other webpack plugins do. We don't support anything less than 3 so it is good if that warning gets printed.

Examples:
https://github.com/webpack-contrib/extract-text-webpack-plugin/blob/master/package.json#L65
https://github.com/babel/babel-loader/blob/master/package.json#L20